pkg/manifests/yaml/sched/controller/rolebinding.yaml

```diff
@@ -0,0 +1,13 @@
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: topology-aware-controller-as-kube-controller
+  namespace: kube-system
+subjects:
+  - kind: ServiceAccount
+    name: topology-aware-controller
+    namespace: tas-scheduler
```

Comment:
do we need to adjust the namespace(s) here? - `pkg/manifests/updates.go`

Reply:
It can be part of ```kube-system``` ns, but it's not mandatory I think.
I don't see ```pkg/manifests/updates.go``` file under this rep, what am I missing here?